tools/CMakeLists.txt

```diff
@@ -43,29 +39,28 @@ if(ENABLE_PYTHON)
                                                    ${PYTHON_BRIDGE_OUTPUT_DIRECTORY}/tools/benchmark
     )
 
-    ie_cpack_add_component(python_tools_${PYTHON_VERSION})
-    ie_cpack_add_component(python_tools)
-
-    # install benchmark_app tool
+    # install BA tool for tests
     install(FILES benchmark_tool/README.md
                   benchmark_tool/requirements.txt
             DESTINATION tools/benchmark_tool
-            COMPONENT python_tools)
+            COMPONENT tests
+            EXCLUDE_FROM_ALL)
 
     install(PROGRAMS benchmark_tool/benchmark_app.py
             DESTINATION tools/benchmark_tool
-            COMPONENT python_tools)
+            COMPONENT tests
+            EXCLUDE_FROM_ALL)
 
     # install openvino/tools/benchmark as a python package
     install(DIRECTORY benchmark_tool/openvino/tools/benchmark
             DESTINATION python/${PYTHON_VERSION}/openvino/tools
             USE_SOURCE_PERMISSIONS
-            COMPONENT python_tools_${PYTHON_VERSION})
-
-    ie_cpack(python_tools python_tools_${PYTHON_VERSION})
+            COMPONENT tests
+            EXCLUDE_FROM_ALL)
+endif()
```

Comment:
I believe it finally should be removed, instead of making as `tests`. Benchmark app should be deployed by opebvino-dev wheel for test purposes.